WebRoot/js/zimbraMail/share/model/ZmSettings.js

```diff
@@ -930,6 +930,7 @@ function() {
 	this.registerSetting("EXPORT_MAX_DAYS",					{name:"zimbraExportMaxDays", type:ZmSetting.T_COS, dataType:ZmSetting.D_INT, defaultValue:0});
 	this.registerSetting("FILTER_BATCH_SIZE",               {name:"zimbraFilterBatchSize", type:ZmSetting.T_COS, dataType:ZmSetting.D_INT, defaultValue: 10000});
     this.registerSetting("FLAGGING_ENABLED",				{name:"zimbraFeatureFlaggingEnabled", type:ZmSetting.T_COS, dataType:ZmSetting.D_BOOLEAN, defaultValue:true});
+	this.registerSetting("ENABLE_RETENTION_POLICY",			{name:"zimbraFeatureRetentionPolicyEnabled", type:ZmSetting.T_PREF, dataType:ZmSetting.D_BOOLEAN, defaultValue:false});
```

Comment:
Default value should be true.
https://github.com/Zimbra/zm-mailbox/pull/1498
```
<attr id="4101" name="zimbraFeatureRetentionPolicyEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited" since="10.1.0">
  <defaultCOSValue>TRUE</defaultCOSValue>
  <defaultExternalCOSValue>FALSE</defaultExternalCOSValue>
  <desc>Whether retention policy feature is enabled</desc>
</attr>
```

Reply:
Done.